src/App.jsx

```diff
@@ -19,6 +19,24 @@ class App extends React.Component {
   }
 
   render() {
+    var lock = new Auth0Lock('KvwsETaUxuXVjW2cmz2LbdqXQBdYs6wH', 'loom.auth0.com');
```

Comment:
you'll need something like this...

    import Auth0Lock from 'auth0-lock';

it depends on what they're exporting